### ISSUE TITLE
Kite doesn't recognize atom package on ubuntu

### ISSUE THREAD

**Issue by Actanton** [kiteco/issue-tracker#453]
When opening the kite application on ubuntu 20.10, the application doesn't recognize the atom package. In atom, the kite logo displays a message indicating that it is "ready" and I am getting suggestions but the application isn't connected so the AI learning aspect of Kite isn't working.
https://s3.console.aws.amazon.com/s3/object/kite-client-logs/prod/529362/kite_local/client.log.2020-10-29_09-48-12-PM.bak.gz

**Comment by its-dhung** [comment]
Do you mean that within the Copilot's plugin manager, Kite does not recognize Atom as being installed?

**Comment by Actanton** [comment]
Yes, that's exactly what I mean. When I go to the plug-ins tab of the copilot, it says that it doesn't recognize the atom plugin as being installed. I've tried following the directions provided on the links for troubleshooting but to no avail.

**Comment by its-dhung** [comment]
This shouldn't actually affect any of Kite's functionality within Atom. It's more a bug with how our plugin manager works (specifically on Linux in your case). Where on your system is Atom installed?
